ui/components/src/BarChart/BarChart.tsx

```diff
@@ -66,8 +66,14 @@ export function BarChart(props: BarChartProps) {
       series: {
         type: 'bar',
         label: {
+          show: true,
+          position: 'right',
```

Comment:
Moving this back to BarChart since wasn't thinking about how it would impact vertical bar charts when I suggested adding it to chartsTheme in #1279 (sorry @ellenwyllie!). And moving color back as well so that TimeSeriesChart stacked bar loops through the palette again